recipe/build-python.sh

```diff
@@ -11,6 +11,8 @@ cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
       -DBUILD_SAMPLES=OFF \
       -DBUILD_EXAMPLES=OFF \
       -DBUILD_PYTHON=ON \
+      -DFETCH_PYTHON_DEPS=OFF \
+      -DENABLE_TEST=OFF \
```

Comment:
should be `-DBUILD_TESTING=OFF` my bad
note: `enable_test()` is the name of the internal method to active them